StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -1175,3 +1183,37 @@ bool StPicoDstMaker::selectVertex()
   // Retrun false if selected vertex is not valid
   return selectedVertex ? true : false;
 }
+
+void StPicoDstMaker::fillFmsHits()
+{
+  /*
+  StFmsDbMaker* fmsDb=static_cast<StFmsDbMaker*>(GetMaker("fmsDb"));//can save this into a member variable
+  if(!fmsDb){
+    LOG_ERROR<<"fillFmsHits::Failed to get StFmsDbMaker"<<endm;
+    return;
+  }
+  */
+
+  StEvent* event = (StEvent*) GetInputDS("StEvent");
+  if(!event) {
+    LOG_ERROR << "No StEvent found" << endm;
+    return;
+  }
+  StFmsCollection* fmsColl = event->fmsCollection();
```

Comment:
Don't we have the same collection available in muDst branches? If yes, get it from muDst instead of StEvent